### DISCUSSION TITLE
Build system broken

### DISCUSSION THREAD

**Discussion by lolodomo** [openhab/openhab-addons#14709] [Add-on Maintainers Discussions]
Hi all.
We have serious difficulties with the build system.
All PR builds are failing relatively quickly, often due a JVM exit code 137.
Is there someone looking at this problem ?
Until it is fixed, can we at least merge small PRs with no risk (like for example only strings updated) ? Or should we wait until the build of each PR is green ?
Have a good day

**Comment by fwolter** [comment]
If it builds locally, it's supposed(!) to build on Jenkins, too. I merge PRs also if the CI build fails, but succeed locally. We had issues with several unstable tests in the past, but that's a decoupled problem and may be fixed in separate PRs, IMHO.

We use this only for organisational topics. This is the current thread for all other topics https://community.openhab.org/t/a-very-new-addons-maintainer-mail/126614/4
